website/docs/api/cli.mdx

```diff
@@ -249,6 +251,93 @@ $ python -m spacy init labels [config_path] [output_path] [--code] [--verbose] [
+| `output_file`      | Path to output `.cfg` file or `-` to write the config to stdout (so you can pipe it forward to a file or to the `train` command). Note that if you're writing to stdout, no additional logging info is printed. ~~Path (positional)~~ |
+| `transformer_name` | The name of the base HuggingFace model to use. Defaults to `roberta-base`. ~~str (option)~~                                                                                                                                           |
+
+### configure tok2vec {id="configure-tok2vec", tag="command"}
+
+Modify the base model config to use a CNN tok2vec component. Useful for
+generating a config from a transformer-based model for faster training
+iteration.
+
+```cli
+$ python -m spacy configure tok2vec [base_model] [output_file]
+```
+
+| Name          | Description                                                                                                                                                                                                                           |
+| ------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `base_model`  | A trained pipeline to resume training (package name or path). ~~str (positional)~~                                                                                                                                                    |
+| `output_file` | Path to output `.cfg` file or `-` to write the config to stdout (so you can pipe it forward to a file or to the `train` command). Note that if you're writing to stdout, no additional logging info is printed. ~~Path (positional)~~ |
+
+## merge {id="merge", tag="command"}
+
+Take two pipelines and create a new one with components from both of them,
+handling the configuration of listeners. The output is a serialized pipeline.
+
+Components in the final pipeline are in the same order as in the original
+pipelines, with the base pipeline first and the added pipeline after. Because
+pipeline names must be unique, if there is a name collision in components, the
+later components will be automatically renamed.
+
+For components with listeners, the resulting pipeline structure depends on the
+number of listeners. If the second pipeline has only one listener, then
+[`replace_listeners`](https://spacy.io/api/language/#replace_listeners) will be
+used. If there is more than one listener, `replace_listeners` will not be used.
+In the multi-listener case, the resulting pipeline may require more adjustment
+for training to work.
+
+This is useful if you have trained a specialized component, such as NER or
+textcat, and want to provide with one of the official pretrained pipelines or
```

Comment:
Same here, dunno about formatting.
```suggestion
`textcat`, and want to provide with one of the official pretrained pipelines or
```

Reply:
I feel like this is also being used in a generic sense, and "NER or `textcat`" is weird, but it does seem that it's less common to use this unmarked than "tok2vec", though there are exceptions like [here](https://github.com/explosion/spaCy/blob/03a0c2badcc3e0355d52110fa0c526e8bbb52324/website/docs/api/language.mdx?plain=1#L260).

Reply:
In line with the previous answer - I think you made the right call.